imperial_coldfront_plugin/views.py

```diff
@@ -35,6 +37,36 @@
 User = get_user_model()
 
 
+@login_required
+@permission_required(
+    "imperial_coldfront_plugin.add_researchgroup", raise_exception=True
+)
```

Comment:
We're not using the Django model level permissions so this isn't relevant. Instead the view should take the username of the user making the request and pull the users profile from the microsoft graph api. Then it should check the profile data with the `user_eligible_to_be_pi` function from `policy.py`. If the profile data fails the check it should deny permission.

Reply:
This is still outstanding.